packages/sitecore-jss/src/feaas/index.ts

```diff
@@ -0,0 +1 @@
+export * from './utils'
```

Comment:
Missing semicolon and newline (check linting - this is currently failing in PR check)